deeplake/core/vectorstore/vector_search/utils.py

```diff
@@ -19,6 +19,19 @@ def parse_tensor_return(tensor):
     return tensor.data(aslist=True)["value"]
 
 
+def parse_exec_option(dataset, exec_option):
+    if exec_option == "auto":
+        if "hub://" in dataset.path:
+            if "vector_db/" in dataset.base_storage.path:
```

Comment:
Checking for wether dataset is in tensordb definitely works. Path is unlikely to change at the moment so should be good for now.